### DISCUSSION TITLE
Upgrade from 2.0 to 3.0 Missing Notification Icon

### DISCUSSION THREAD

**Discussion by bcollins-oxford** [filamentphp/filament#8348] [General]
For the life of me, I cannot figure out where it's trying to load this icon.

"Svg by name "o-inbox-in" from set "heroicons" not found." which only occurs when I add ->databaseNotifications() to the AdminPanelProvider.php

Anyone have any ideas on how I can fix this?

**Comment by danharrin** [comment]
It'll be in one of the database notifications (in the database_notifications table) you sent before upgrading, you'll need to migrate those to use the new heroicon names

**Reply by bcollins-oxford** [reply]
Ahh!  That was it, all of the unread notifications I had in my dev environment had that icon assigned.
